### PR TITLE
AAP-22284: Created docs for playbook generation and explanation feature

### PR DESCRIPTION
**Pull request by sayjadha**
This PR addresses creates docs for 'generating playbooks and viewing playbook explanations' feature.

JIRA: https://issues.redhat.com/browse/AAP-22284.

Changes made:

- Created new files for playbook generation and playbook explanations features.
- Added illustrations wherever necessary
Note: The release notes blurb will be created in a separate ticket, as I'm working on another incoming feature Lightspeed on-premise deployment, which will be released around the same time.

### CODE REVIEW

File: lightspeed/assemblies/assembly_generating-playbooks.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+[id="generating-playbooks_{context}"]
+
+= Generating playbooks and viewing playbook explanations
+
+:context: generating-playbooks
+
+[role="_abstract"]
+
+Using the Ansible VS Code extension, you can create Ansible playbooks using a natural language interface in English. {LightspeedshortName} reads the natural language prompts, identifies the context for the playbook, and generates an entire playbook recommendation based on your task intent. You can also view the explanations for new or existing playbooks. The playbook explanations describe what the playbook will do and contextualize its impact.
```
Comment (ianf77) [inline lightspeed/assemblies/assembly_generating-playbooks.adoc:13]: what the playbook does and contextualizes its impact.

File: lightspeed/modules/proc_generate-playbook.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+You can use the natural language interface in the Ansible VS Code extension to generate an entire Ansible playbook.
+
+.Prerequisites
+
+* You are part of an organization that has a trial or paid subscription to {PlatformName} and {ibmwatsonxcodeassistant} both.
+* You have installed and configured the Ansible VS Code extension. 
+
+.Procedure
+
+. Log in to VS Code with your Red Hat account.
```
Comment (ianf77) [inline lightspeed/modules/proc_generate-playbook.adoc:16]: Is it VS Code or VSCode?

**Review by ianf77 — COMMENTED** [review]
A few minor comments.

File: lightspeed/assemblies/assembly_generating-playbooks.adoc (hunk 1 above)
Comment (sayjadha) [reply lightspeed/assemblies/assembly_generating-playbooks.adoc]: This is done, thank you

File: lightspeed/modules/proc_generate-playbook.adoc (hunk 2 above)
Comment (sayjadha) [reply lightspeed/modules/proc_generate-playbook.adoc:16]: It is VS Code. See https://code.visualstudio.com/.

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**